docs/articles_en/assets/snippets/ov_caching.py

```diff
@@ -8,7 +8,8 @@
 
 import openvino.properties as props
 
-device_name = 'CPU'
+# For example: "CPU", "GPU", "NPU".
+device_name = 'GPU'
```

Comment:
```suggestion
device_name = 'CPU'
```
Reverting value due to CI tests.